### PR TITLE
Add nifi registry support

### PR DESCRIPTION
**Pull request by daniel-yavorovich**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by alexnuttinck** [comment]
Hello @daniel-yavorovich, 
Thanks for your work on nifi-registry!
I will take a bit of time to see if it integrates well as a dependency, but I think it is a good idea.
A first remark would be that there is no README on your repo https://github.com/dysnix/charts/tree/master/nifi-registry and would be great to have one.

File: Chart.yaml [hunk 1]
```diff
@@ -16,8 +16,15 @@ maintainers:
   - name: alexnuttinck
     email: alexandre.nuttinck@cetic.be
     url: https://github.com/alexnuttinck
+  - name: daniel-yavorovich
```
Comment (alexnuttinck) [inline Chart.yaml:19]: For the moment, I can't add you as a maintainer of the Chart, you will be maintainer of the nifi-registry Chart :) I can consider to add you in the case you participate in bug fixes, feature implementations...

Comment (daniel-yavorovich) [reply Chart.yaml]: sure, I have removed it. Thank you for explain!

**Comment by alexnuttinck** [comment]
Thanks @daniel-yavorovich, I merge your PR, it will present in the next release. I will just add an alias for the dependency.

**Pull request merged**